app/seeders/user_seeder.rb

```diff
@@ -0,0 +1,18 @@
+class UserSeeder
+  USERS = [
+    {
+        email: "name@gmail.com",
+        password: "password",
+        password_confirmation: "password"
+    }, {
+        email: "someone@gmail.com",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.